### DISCUSSION TITLE
Welcome to mailbook Discussions!

### DISCUSSION THREAD

**Discussion by Xammie** [Xammie/mailbook#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by pauleniz** [comment]
Hello is this package only for dev env? I'd like to use it in production to preview emails and their translations. I tried to give the middlewares auth and is_admin in config file but it didnt work.

**Reply by Xammie** [reply]
Hello! Yes this can be used in production. In fact I am using it in production for a project.

You can define middlewares in the `mailbook.php` config file

```php
'middlewares' => [
        'web',
        \App\Http\Middleware\MailbookMiddleware::class, // this is my custom
        Xammie\Mailbook\Http\Middlewares\RollbackDatabase::class,
],
```

In this example i've added a new custom middleware called MailbookMiddleware. The middleware will check if the user is authorized to access mailbook. Let me know if you still have any questions.

**Reply by Xammie** [reply]
@pauleniz Also I forgot to mention that you have to change the `enabled` config option to

```php
'enabled' => true,
```

You can see in the [default config](https://github.com/Xammie/mailbook/blob/492fd02ffcbfded18e43b0932bf1d838e1b649fe/config/mailbook.php) that it is disabled for any environment other than `local`
